tests/generate_fixtures/generate_target_dnadiff_files.py

```diff
@@ -58,6 +60,14 @@
 SHOW_DIFF_DIR = Path("../fixtures/dnadiff/targets/show_diff")
 SHOW_COORDS_DIR = Path("../fixtures/dnadiff/targets/show_coords")
 
+# dnadiff generates many files; since we only need `.report` files,
+# we will save all files to a temporary location, move the report files
+# to a specific directory, and remove the rest.
+Path("temp_dnadiff").mkdir(parents=True, exist_ok=True)
+
+DNADIFF_DIR = Path("../fixtures/dnadiff/targets/dnadiff_reports")
+DNADIFF_TEMP_DIR = Path("temp_dnadiff")
+
```

Comment:
Maybe use ``import tempfile`` and something like ``generate_anib_blast_files.py`` does for a temp directory?

Reply:
I have changed it now. Thanks for introducing me to `tempfile`!